src/Sylius/Bundle/OrderBundle/Form/DataMapper/OrderItemQuantityDataMapper.php

```diff
@@ -65,7 +65,11 @@ public function mapFormsToData($forms, &$data)
         }
 
         if (!empty($formsOtherThanQuantity)) {
-            $this->propertyPathDataMapper->mapFormsToData($formsOtherThanQuantity, $data);
+            try {
+                $this->propertyPathDataMapper->mapFormsToData($formsOtherThanQuantity, $data);
+            } catch (\InvalidArgumentException $ex) {
```

Comment:
That's how I avoid getting 500 Internal Server Error, because this will call OrderItem::setUnitPrice(null) when not POST-ing unitPrice. Another way of fixing this is not adding null values to $formsOtherThanQuantity in the code above.